tools/backups2datalad/__main__.py

```diff
@@ -150,6 +150,8 @@ def update_from_backup(
     exclude: Optional[re.Pattern[str]],
     tags: bool,
 ) -> None:
+    if (gh_org is None) != (zarr_gh_org is None):
```

Comment:
could be just
```suggestion
    if bool(gh_org) != bool(zarr_gh_org):
```
but since I don't think we would have empty strings, effect is the same so ok with me as is